test/uk/gov/hmrc/gform/eval/EvaluationResultsSpec.scala

```diff
@@ -357,9 +357,9 @@ class EvaluationResultsSpec extends Spec with TableDrivenPropertyChecks {
   it should "evaluate form component reference from outside, when they are in indexed pages (repeating page, ATL)" in {
 
     val table = Table(
-      ("typeInfo", "recData", "evaluationContext", "expectedResult", "scenario"),
+      ("typeInfo", "recData", "evaluationContext", "expectedResult", "exprMap", "scenario"),
```

Comment:
they are not longer working with empty `exprMap` because when we evaluate to `ListResult` we use only `exprMap` but previously we got it from `recData`

Reply:
Ok, that make sense